RELEASES.md

```diff
@@ -2,6 +2,256 @@
+configuration file). The `ToS` value is encoded in the `IPv4` header and is comprised of the Differentiated services
+code point (`DSCP`) and the Explicit congestion notification (`ECN`) fields.
+
+Setting the `ToS` on outgoing probe packets can influence the Quality of Service (`QoS`) used by the network devices
+along the path.
+
+Probe responses received from the hops along the path include the `ToS` values in the Original Datagram (the `IPv4`/
+`IPv6` header of the probe packet nested inside the `ICMP` error). Examining the `ToS` value from the Original Datagram
+can provide a useful insight into how network devices along the path are treating the probe packets.
+
+This release of Trippy adds two new column to display the `DSCP` & `ECN` values derived from the `ToS` value from the
+Original Datagram for each hop:
+
+- `Dscp` (`K`): The Differentiated Services Code Point (`DSCP`) of the Original Datagram for a hop
+- `Ecn` (`M`): The Explicit Congestion Notification (`ECN`) of the Original Datagram for a hop
+
+The `Dscp` and `Ecn` columns are decoded from the `ToS` field of the Original Datagram. If no `ToS` value is present
+then the columns will be shown `na`. Note that these columns show the most recent `ToS` value received from the hop and
+may therefore change between rounds.
+
+Well-known `DSCP` values are displayed as follows:
+
+- Default Forwarding (`DF`) aka Best Effort aka Class Selector 0 (`CS0`)
+- Assured Forwarding (`AFn`)
+- Class Selector (`CSn`)
+- High Priority Expedited Forwarding (`EF`)
+- Voice Admit (`VA`)
+- Lower Effort (`LE`)
+
+Unknown `DSCP` values are displayed as a hexadecimal value.
+
+The `ECN` value will be displayed as follows:
+
+- Not ECN-Capable Transport (`NotECT`)
+- ECN Capable Transport(1) (`ECT1`)
+- ECN Capable Transport(0) (`ECT0`)
+- Congestion Experienced (`CE`)
+
+These columns are hidden by default but can be enabled as needed. For more details, see
+the [Column Reference](https://trippy.rs/reference/column).
+
+The following screenshot shows an example trace with the new columns enabled:
+
+** TODO screenshot **
+
+The ToS field of the Original Datagram has also been added to the `json` output format as a decimal value.
+
+See [#1539](https://github.com/fujiapple852/trippy/issues/1539) for details.
+
+This release also adds support for setting the `IPv6` Traffic Class (which encodes the `DSCP` & `ECN` values in the same
+way as `IPv4`) via the same `--tos` (`-Q`) command-line argument (or via the configuration file). Note that setting the
+`IPv6` Traffic Class is not currently supported on Windows.
+See [#202](https://github.com/fujiapple852/trippy/issues/202) for details.
+
+Finally, a bug which caused the `--tos` (`-Q`) command-line argument to be ignored for `IPv4/UDP` tracing has been
+fixed in this release. See [#1540](https://github.com/fujiapple852/trippy/issues/1540) for details.
```

Comment:
@moeller0 can you please review this section of the release note?

Reply:
Looks correct. Thanks. Sorry for the delay, spend some time away from the keyboard...